_posts/2019-02-04-lecture-06.md

```diff
@@ -0,0 +1,503 @@
+</figure>
+
+#### Example: A Baysian Network for Biological Evolution
+<figure>
+<img src="{{ '/assets/img/notes/lecture-06/lec6_bioEvol.png' | relative_url }}" />
+<figcaption>
+<strong> This Baysian Network is not full observable </strong>. We cannot measure data from evolutionary ancestors no longer on earth, but they are still a useful latent variable for the model.
+</figcaption>
+</figure>
+
+### Probabilistic Inference
+A GM $M$ describes a unique probability distribution $P$.  To learn partially observable GMs, we must iterate between inferring relationships described by $P$ and estimating a plausible model $M$ from data $D$. In this way *inference* can be considered a subroutine of the learning problem:
+
+**Task 1.** How do we answer queries about $P$, e.g. $P(X|Y)$? 
+- *We use inference as a name for the process of computing answers to these queries.*
+
+**Task 2.** How do we estimate a plausible model $M$ from data $D$?
+- *We use learning as a name for the process of a obtaining point estimate of $M$*. 
+*For the Bayesian approach, we seek $P(M|D)$ , which is also an inference problem.*
+
+There are many approaches for inference in GMs. They can be divided into two classes:
++ **Exact inference algorithms.** Including the elimination algorithm, message-passing algorithm (sum-product, belief propagation), the junction tree algorithms. These algorithms can give the precise result of query. The major topic of this lecture is on exact inference algorithms.  
++ **Approximate inference techniques.** Including stochastic simulation / sampling methods, Markov chain Monte Carlo (MCMC) methods, variational algorithms. These algorithms only gives an approximate answer to the inference query. We will cover these methods in future lectures.
+
+
+### Mixture Models
+A density model $p(x)$ may be multi-modal, but we may be able to model it as a mixture of uni-modal distributions (e.g. Gaussians). Suppose we have a dataset $x_n \in \mathbb{R}^D$ where $n=1$,...,$N$, and we believe the data has been generated from a mixture of $K$ Gaussian components.  Let $z \in \{1,...,K \}$ be an *unobserved* random variable, such that $P(z = k) = \pi_k$, where $k = 1,...,K$. For $P(z)$ to be a valid probability distribution, $0 \leq \pi_k \leq 1$ and $\sum_{k=1}^K \pi_k =1$. We call $\pi_k$ the *mixture proportion*. Our assumption of the Gaussian components given a mixture label $k$ is expressed as $P(x| z=k) = N(x |\mu_k, \Sigma_k)$, where $\mu_k$ and $\Sigma_k$ are the parameters for each Gaussian component. Thus,
+
+$$ 
+p(x_n) = \sum_z P(x | z) P(z) = \sum_{k=1}^K N(x | \mu_k, \Sigma_k) \pi_k 
+$$
+
++ $Z$ is a latent class indicator vector:
+
+$$ 
+p(z_n) = \text{multi}(z_n | \pi) = \prod_k (\pi_k)^{z_n^k} 
+$$
+
++ $X$ is a conditional Gaussian variable with a class-specific mean/covariance:
+
+$$ 
+p(x_n | z_n^k = 1, \mu, \Sigma) = \frac{1}{(2\pi)^{m/2} |\Sigma_k|^{1/2}} \text{exp} (-\frac{1}{2} (x_n - \mu_k)^T \Sigma_k^{-1} (x_n - \mu_k))
```

Comment:
please use \left( and \right)

Reply:
Hard to make sure everything matches without newcommand. Just added macros to help in katex.js since gdef doesn't seem to be working correctly. Now `\abs{\Sigma}` will give you properly sized vertical bars, `\norm` for double bars, `\parens` for parens, etc. If we put some basic defines in there and document them it could make scribing easier.